skype/skype-ps/skype/New-CsAutoAttendantMenuOption.md

```diff
@@ -43,12 +43,21 @@ $menuOption = New-CsAutoAttendantMenuOption -Action TransferCallToTarget -DtmfRe
 
 This example creates a menu option to transfer the call to an application endpoint when the caller speaks the word "Sales" or presses the 1 key.
 
+### -------------------------- Example 3 --------------------------
+```
+$Prompt = New-CsAutoAttendantPrompt -ActiveType -TextToSpeechPrompt "Our Office is open from Monday to Friday from 9 AM to 5 PM"
```

Comment:
Please check the cmdlet as I received an error:
`New-CsAutoAttendantPrompt : Missing an argument for parameter 'ActiveType'. Specify a parameter of type
'System.Object' and try again.`

Reply:
```suggestion
$Prompt = New-CsAutoAttendantPrompt -ActiveType TextToSpeech -TextToSpeechPrompt "Our Office is open from Monday to Friday from 9 AM to 5 PM"
```
Agree with @get-itips, I think the value for ActiveType should be TextToSpeech.